pkg/system/prlimit.go

```diff
@@ -97,6 +98,10 @@ func scanLinesWithCR(data []byte, atEOF bool) (advance int, token []byte, err er
 func processScanner(scanner *bufio.Scanner, buf *bytes.Buffer, done chan bool, callback func(string)) {
 	for scanner.Scan() {
 		line := scanner.Text()
+		//improve log for better readability:
+		if strings.HasPrefix(line, "qemu-img:") {
+			buf.WriteString("\n")
```

Comment:
Okay, I think you can write the newline after writing the line regardless of what the prefix is.

Reply:
But then the entire progress (percentage) is printed line by line...

Reply:
Right, but that is how it happens in the normal logs as well. I think we would want everything printed line by line, if qemu prints it line by line no?

Reply:
no problem.